packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -65,13 +65,15 @@ const IMPLICIT_DEFAULT_HTML_PATH = '@lwc/resources/empty_html.js';
 const EMPTY_IMPLICIT_HTML_CONTENT = 'export default void 0';
 const IMPLICIT_DEFAULT_CSS_PATH = '@lwc/resources/empty_css.css';
 const EMPTY_IMPLICIT_CSS_CONTENT = '';
+/** Matches all permutations of CJS/ESM, JS/TS, JS/JSX. */
+const SCRIPT_FILE_EXTENSION_REGEX = /^\.[cm]?[jt]sx?$/i;
```

Comment:
Can we simplify this to something that uses an explicit list of extensions that doesn't require a comment?

Reply:
Also, if we're going to support `.cjs`/ `.mjs` / etc., we should add it to this list as well:

https://github.com/salesforce/lwc/blob/c9aa701f3557cb7160a8ab86fc3772289821241a/packages/%40lwc/compiler/src/transformers/transformer.ts#L125-L128

Reply:
Plus, I don't think we support `.cjs` anyway? The `@lwc/babel-plugin-component` will explode if it encounters CommonJS (e.g. `const { LightningElement } = require('lwc')`).